spec/spec_helper.rb

```diff
@@ -78,7 +78,7 @@ def setup_fluent_logger
   end
 end
 
-def setup_before_after
+def setup_before_and_after_for_fluent_logger_spec
```

Comment:
I don't think anything changed. This still does not describe what it does at all.